stable_baselines3/common/vec_env/base_vec_env.py

```diff
@@ -1,12 +1,18 @@
 import inspect
 from abc import ABC, abstractmethod
-from typing import List, Optional, Sequence, Union
+from typing import Any, Dict, Iterable, List, Optional, Sequence, Tuple, Union
 
 import cloudpickle
+import gym
 import numpy as np
 
 from stable_baselines3.common import logger
 
+# Avoid circular import
+VecEnvIndices = Union[None, int, Iterable[int]]
```

Comment:
These could use bit of docs on what they will contain (_e.g._ VecEnvStepReturn contains obs, rewards, dones and infos).